src/journal/journald-client.c

```diff
@@ -0,0 +1,106 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "cgroup-util.h"
+#include "errno-util.h"
+#include "journald-client.h"
+#include "nulstr-util.h"
+#include "pcre2-util.h"
+
+/* This consumes both `allow_list` and `deny_list` arguments. Hence, those arguments should be considered
+ * invalid after calling this function. */
```

Comment:
Not a matter, but the arguments will _not_ become invalid, but should not be freed.

Reply:
I'm not saying those are invalid, but that they _should be considered_ invalid by the caller. I'll update the comment to clarify.